### PR TITLE
Add L3 risks rosettes 

### PR DESCRIPTION
**Pull request by sdlyy**
Resolves L2B-5272

### CODE REVIEW

File: packages/frontend/src/app/(top-nav)/scaling/projects/[slug]/_components/rosette-selector.tsx [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+        <RadioGroupItem value="individual">Individual</RadioGroupItem>
+      </RadioGroup>
+      <Tooltip>
+        <TooltipTrigger>
+          <InfoIcon />
+        </TooltipTrigger>
+        <TooltipContent>
+          {/* FIXME */}
+          Switch between combined risks and individual risks for given L3 and
+          Host Chain.
```
Comment (radinat) [inline packages/frontend/src/app/(top-nav)/scaling/projects/[slug]/_components/rosette-selector.tsx:42]: Change text to: The combined view shows the L3 risk (based on both the L2 and the L3). The individual view displays the L2 and L3 risks separately.

File: packages/frontend/src/components/projects/sections/l3-risk-analysis-section.tsx [hunk 2]
```diff
@@ -0,0 +1,286 @@
...
+    l2.risks
+      .concat(l3.risks)
+      .some(({ sentiment }) => sentiment === 'UnderReview')
+
+  return (
+    <ProjectSection {...sectionProps} isUnderReview={isUnderReview}>
+      <div className="text-sm dark:text-white/80">
+        The L3 risk rosette depends on the individual L3 properties and the
+        risks of the host chain. The combined view shows the risk based on both
+        the L2 and the L3.
```
Comment (radinat) [inline packages/frontend/src/components/projects/sections/l3-risk-analysis-section.tsx:56]: Change text to: The L3 risks depend on the individual properties of L3 and those of the host chain combined.

**Review by radinat — CHANGES_REQUESTED** [review]

File: packages/frontend/src/app/(top-nav)/scaling/projects/[slug]/_components/rosette-selector.tsx (hunk 1 above)
Comment (sdlyy) [reply packages/frontend/src/app/(top-nav)/scaling/projects/[slug]/_components/rosette-selector.tsx]: https://github.com/l2beat/l2beat/pull/5076/commits/5d28f633b273c4a9e0d64fb2e631ead3c7d81651

File: packages/frontend/src/components/projects/sections/l3-risk-analysis-section.tsx (hunk 2 above)
Comment (sdlyy) [reply packages/frontend/src/components/projects/sections/l3-risk-analysis-section.tsx]: https://github.com/l2beat/l2beat/pull/5076/commits/5d28f633b273c4a9e0d64fb2e631ead3c7d81651

**Comment by lucadonnoh** [comment]
image

i have a feeling that the Combined/Individual switch should be right aligned rather than left aligned, wdyt?

**Review by radinat — APPROVED** [review]

**Review by imxeno — APPROVED** [review]

**Pull request merged**